src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int RADIUS = 10;
+    private Random random = new Random();
+
+    public Figure getRandomFigure() {
+        int figureType = random.nextInt(4); // 4 different type of figures
```

Comment:
You should define a constant for the magic number used in `nextInt` call to make it clear what it represents. According to the checklist, all magic numbers should be constants.